src/amo/components/CategoryHead/index.js

```diff
@@ -64,9 +53,13 @@ export class CategoryHeadBase extends React.PureComponent<InternalProps> {
       <React.Fragment>
         <Helmet>
           <title>{this.getPageTitle()}</title>
-          {this.renderMetaDescription()}
         </Helmet>
 
+        <HeadMetaTags
+          description={category.description}
+          title={this.getPageTitle()}
```

Comment:
maybe this could be set to a var since it's used more than once..?

Reply:
yup